src/Components/spoiler/spoiler.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import "./spoiler.css";
+
+
+
+export default class Spoiler extends Component{
```

Comment:
You don't need class components anymore. Only if you want to create ErrorBoundary but it's outside of the scope.